flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateWriteRequestExecutorImplTest.java

```diff
@@ -128,12 +131,33 @@ public void testIgnoresInterruptsWhileRunning() throws Exception {
         }
     }
 
-    @Test
-    public void testCanBeClosed() throws IOException {
-        TestRequestDispatcher requestProcessor = new TestRequestDispatcher();
+    @Test(timeout = 10000)
```

Comment:
```suggestion
    @Test
```
We don't use timeouts in tests

Reply:
Why don't we use timeout in tests? For old code, if don't use timeout, it will be stuck forever, so I add the timeout.

Reply:
We have agreed to be fine with that risk. We've had several cases where tests started to fail because certain CI mechanisms triggered the timeout (e.g., time skips).

Reply:
Hi @zentol @pnowojski , thanks for your clarification. I removed it.